covid_graphs/setup.py

```diff
@@ -18,6 +18,7 @@
         "plotly",
         "pytest",
         "scipy",
+        "uwsgi"
```

Comment:
I don't think that the package actually uses `uswgi` anywhere. Also, it shouldn't.

Reply:
uwsgi package is standard uwsgi wrapper with python support. On one side, it talk to webserver (nginx) using wsgi protocol, on the other side it translates all calls to python calls using flask.

There is more straighforward option. Just use flask, let it listen on some port using http protocol and forward every call tu it from nginx. I used uWsgi because it was mentioned in #58.

Reply:
You're missing the point. This is a list of Python dependencies. This package doesn't use the `uwsgi` - it's not dependent on it.

Reply:
So proper way is to install it independantly during setup in Dockerfile. Right?

Reply:
Yes